wgkex/worker/netlink.py

```diff
@@ -179,7 +179,10 @@ def find_stale_wireguard_clients(wg_interface: str) -> List:
         (datetime.now() - timedelta(hours=_PEER_TIMEOUT_HOURS)).timestamp()
     )
     with pyroute2.WireGuard() as wg:
-        clients = wg.info(wg_interface)[0].WGDEVICE_A_PEERS.value
+        clients = []
+        infos = wg.info(self.wg_interface)
```

Comment:
```suggestion
        infos = wg.info(WireGuardClient.wg_interface)
```

Reply:
This should fix it.

Reply:
sure you want it this way? see my latest commit for my approach, which just uses `wg_interface` (the functions parameter)

Reply:
weird. This looked different a second ago.
We now have the same solution which is already part of this now open PR.